article.html

```diff
@@ -24,24 +24,17 @@ <h2>Author and publish date here</h2>
             </div>
 
             <!-- This div is for similar articles that will be displayed below -->
-            <div class="similar">
+            <div class="similar-articles">

```

Comment:
I removed the flexbox in this version and instead made it a sidebar on the right that has smaller article links and images that the user can click on.